package.json

```diff
@@ -4,9 +4,9 @@
   "description": "Chat bot for destiny.gg chat",
   "main": "index.js",
   "scripts": {
-    "test": "./node_modules/mocha/bin/mocha tests/**/*.test.js --exit --reporter spec",
-    "lint": "./node_modules/.bin/eslint index.js ./lib/**",
-    "start": "node index.js | ./node_modules/bunyan/bin/bunyan"
+    "test": "mocha tests/**/*.test.js --exit --reporter spec",
```

Comment:
So the reason these point to node modules is that in most environments these programs will not be in the user path, and I'd rather not require that they be. I'm not sure why pointing to them with a relative path wouldn't work on windows, but there should be a way to point to their relative paths in the node modules thats OS agnostic.

Reply:
What ever is in `./node_modules/.bin/` is "automagically" added to `PATH`, when running through npm. So the above will NOT run some globally installed mocha, but the one installed in `node_modules`.

>I'm not sure why pointing to them with a relative path wouldn't work on windows

Windows scripts uses a different file entirely ie. `./node_modules/.bin/mocha.cmd` vs `./node_modules/.bin/mocha`

>, but there should be a way to point to their relative paths in the node modules thats OS agnostic

Yes, that is what this change does. :)

Reply:
Good to know! I've been on npm 4 at work so long I didn't even know this was the case.

Reply:
Time to run `npm i -g npm` at work I think 😄